modules/ibi-create-standalone-config-iso.adoc

```diff
@@ -0,0 +1,227 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-edge-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-standalone-config-iso_{context}"]
+= Deploying a standalone {sno} cluster using the {product-title} installation program
+
+For standalone clusters, you can use the {product-title} installation program to configure and deploy a host that you preinstalled with an image-based installation. To configure the target host with site-specific details, you must create the following resources:
+
+* The `install-config.yaml` installation manifest.
+* The `ImageBasedConfig` custom resource.
+
+The `openshift-install` installation program consumes these resources to generate a configuration ISO that you can attach to your preinstalled target host and complete the deployment.
+
+[NOTE]
+====
+For more information about the API specifications for the `ImageBasedConfig` resource, see the _ImageBasedConfig resource API specifications_ section.
```

Comment:
Same as [here](https://github.com/openshift/openshift-docs/pull/79610/files#r1754034359). I think it's better if we just call it `image-based-config.yaml manifest` (or file, whatever you think it's best).